index.js

```diff
@@ -79,6 +82,125 @@ class BitbucketScm extends Scm {
     }
 
     /**
+     * Look for a specific webhook that is attached to a repo.
+     *
+     * Searches through the webhook pages until the given webhook URL is found. If nothing is found, this will
+     * return nothing. If a status response of non-200 is encountered, the chain is rejected with the
+     * HTTP operation and the status code received.
+     * @method _findWebhook
+     * @param  {Object}     config
+     * @param  {Number}     config.page    pagination: page number to search next. 1-index.
+     * @param  {String}     config.repoId  The bitbucket repo ID (e.g., "username/repoSlug")
+     * @param  {String}     config.token   Admin Oauth2 token for the repo
+     * @param  {String}     config.url     url for webhook notifications
+     * @return {Promise}                   Resolves to a webhook information payload
+     */
+    _findWebhook(config) {
+        return this.breaker.runCommand({
+            url: `${API_URL_V2}/repositories/${config.repoId}/hooks?pagelen=30&page=${config.page}`,
+            method: 'GET',
+            login_type: 'oauth2',
+            oauth_access_token: config.token
+        }).then((response) => {
+            if (response.statusCode < 200 || response.statusCode >= 300) {
+                throw new Error(response.statusCode);
```

Comment:
Open to suggestions on how to make this better. Since we're accessing the API over vanilla HTTP requests, we don't have convenient error handling like we do with the `scm-github` module.

This error is still vague, but I couldn't think of anything better during the first pass.